docs/api.rst

```diff
@@ -211,18 +211,72 @@ API Methods
         }
 
 
-   **Description of the JSON response attributes**:
+    **Description of the JSON response attributes**:
 
-   The response contains mappings between the metric names and an array of
-   their timestamp-value pairs, where the pairs in the array are in ascending
-   order of their timestamp values (from the earliest time to the latest time).
-   
-   Each pair contains the timestamp under the ``x`` field, and is formatted as
-   the number of milliseconds elapsed since 1 January 1970 00:00:00 UTC.
+    The response contains mappings between the metric names and an array of
+    their timestamp-value pairs, where the pairs in the array are in ascending
+    order of their timestamp values (from the earliest time to the latest time).
 
-   Each pair contains the value under the ``y`` field, and is formatted as a
-   json number.
+    Each pair contains the timestamp under the ``x`` field, and is formatted as
+    the number of milliseconds elapsed since 1 January 1970 00:00:00 UTC.
 
+    Each pair contains the value under the ``y`` field, and is formatted as a
+    json number.
 
 .. _from and until: http://graphite.readthedocs.org/en/latest/render_api.html#from-until
 .. _functions: http://graphite.readthedocs.org/en/latest/functions.html#graphite.render.functions.summarize
+
+.. http:post:: /api/metrics/
+
+    Fires one or many metrics as specified by the body. Body format is is JSON,
+    in the format of an object of key value pairs, where the key is the name of
+    the metric to fire, and the value is the value to fire for the metric.
+    Multiple metrics may be specified.
+
+    **Example request**:
+
+    .. sourcecode:: http
+
+        POST /api/metrics/ HTTP/1.1
+        Host: www.example.org
+        Content-Type: application/json
+        Authorization: Bearer auth-token
+
+        {
+            "metric1": 27.4,
+            "metric2.sum": 11.2
```

Comment:
I'm not sure if this is a good/bad way, but it does enforce a naming standard for metrics, which could be useful, and it means that names created by the metrics API will be in the same format as the ones created in vumi go.